### ISSUE TITLE
WithSharpFormEloquentUpdater trait doesn't support pivot table fields

### ISSUE THREAD

**Issue by stickgrinder** [code16/sharp#258]
I'm in the condition where I have a pivot table for a many-to-many with `source_id`, `destination_id` and a `condition` field.

In the editing form I can load the value of the condition in text field using ` SharpFormTextField::make("pivot:condition")` but when I save a new value for the field, it is ignored.

Taking a (very) quick look at the mentioned trait, it seems to me that pivotal conditions are not supported in the code.
Is this true? Maybe I can come up with a solution if I have some spare time, but a nudge in the right direction may be appreciated.

Thanks for Sharp, it's a very wonderful piece of software!

**Comment by dvlpp** [comment]
I think you are right, Sharp won't handle this case automatically. I'm not available right now to look for a solution, or even check if there is one in the eloquent auto updater; maybe you can ignore this field in your Form update method, and handle this manually, for now?

**Comment by stickgrinder** [comment]
Hi @dvlpp, thanks for your reply. I actually did what you suggested right after dumping the issue here and obviously, it works like a charm.
I spent some time digging into Sharp's code and I guess I have spotted the right place to support this but to my current understanding it's not possible to introduce an automatic behavior without a decent amount of change at public API level (like explicitly declare a field as pivot and mapping the correct names for the relation.

But I'm not exactly expert with Sharp, and even with Laravel so it may be possible to "guess" those information with a bit of Eloquent magic.

Nothing really bad anyway, ignoring the field and managing it manually is a valid alternative for such a rare case.
Thanks again for your effort, Sharp really is a wonderful piece of software.
